.github/ct.yaml

```diff
@@ -1,5 +1,6 @@
 helm-extra-args: --timeout 120s
 check-version-increment: false
 debug: true
-chart-repos: bitnami=https://charts.bitnami.com/bitnami
+chart-repos:
+  - bitnami=https://charts.bitnami.com/bitnami
```

Comment:
Do we need this helm repos here? I have another repository which contains helm charts and only defines the helm repo url in the Chart.yaml (https://github.com/siticom/helm-charts/blob/main/charts/librenms/Chart.yaml#L30).

Reply:
We need it for testing the charts, I think. And the chart-releaser also seems to depend on it: https://github.com/nimbolus/helm-charts/actions/runs/4545916480/jobs/8013884880#step:4:36

Reply:
Without this line the chart-testing is failing:

```
$ ct install --config .github/ct.yaml --charts charts/terraform-backend
Installing charts...
Version increment checking disabled.
>>> helm version --template {{ .Version }}

------------------------------------------------------------------------------------------------------------------------
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 terraform-backend => (version: "0.1.0", path: "charts/terraform-backend")
------------------------------------------------------------------------------------------------------------------------

>>> helm dependency build charts/terraform-backend
Error: no repository definition for https://charts.bitnami.com/bitnami, https://charts.bitnami.com/bitnami. Please add the missing repos via 'helm repo add'
```

Reply:
I don't understand why your example is not failing, but it seems to be necessary: https://github.com/helm/chart-releaser/issues/135

Reply:
Ha - we just need to update the action...

Reply:
for me the testing job works locally without the `chart-repos` setting. Maybe we should give it a try.

Reply:
Because you have the repo added probably.

Reply:
Ah, the testing job is only triggered for chart changes and the `chart-repos` setting is still required by that action.